scripts/ci/ensure-staged-shrinkwrap.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
I copied from the sass staged files script, which doesn't have a license neither. I added it to both in this PR.